scripts/maint/post-merge.git-hook

```diff
@@ -13,12 +13,20 @@ check_for_diffs() {
                         echo "ATTENTION: $1 hook has changed:"
                         echo "==============================="
                         diff "$installed" "$latest"
-                        exit 1
                fi
         fi
 }
 
+check_for_script_update() {
+        fullpath="$git_toplevel/scripts/maint/$1"
+
+        git diff ORIG_HEAD HEAD --exit-code -- "$fullpath"
```

Comment:
Will `--exit-code` make this script fail if a file has changed?  Maybe we should let the script succeed uniformly, and just warn the user.

On a similar note, we should probably display a helpful message here, and not just a raw diff.

Reply:
Fixed in 92b4d17a64e3dbb6fb6880d9452abdae95f67a5b.